Android/GreenCircle/app/src/main/java/com/greencircle/framework/views/fragments/company/upload_documents/UploadCertificationsFragment.kt

```diff
@@ -0,0 +1,92 @@
+package com.greencircle.framework.views.fragments.company.upload_documents
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.greencircle.R
+import com.greencircle.databinding.FragmentUploadCertificationsBinding
+
+/**Constructor de "UploadCertificationsFragment"
+ *
+ * @constructor Incializa y crea la vista del "UploadCertificationsFragment"
+ */
+class UploadCertificationsFragment : Fragment() {
+    private var _binding: FragmentUploadCertificationsBinding? = null
+    private val binding get() = _binding!!
+
+    /**
+     * Método que se llama cuando se crea la vista del fragmento de subir los documentos de certificaciones.
+     *
+     * @param inflater El inflador de diseño que se utiliza para inflar la vista.
+     * @param container El contenedor en el que se debe colocar la vista del fragmento.
+     * @param savedInstanceState La instancia de Bundle que contiene datos previamente guardados del fragmento.
+     * @return La vista inflada para el fragmento.
+     */
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        // Inflar el diseño de este fragmento
+        _binding = FragmentUploadCertificationsBinding.inflate(inflater, container, false)
+        val root: View = binding.root
+
+        binding.dirInstaladoresCdmxUpload.setOnClickListener {
+            val dialogFragment = UploadDocumentDialogFragment(
+                "Directorio de instaladores certificados de CDMX",
+                "Directorio de instaladores certificados de CDMX",
```

Comment:
Is there a need for a repetitive "Directorio de instaladores certificados de CDMX"?

Reply:
Yes